arcgis-ios-sdk-samples/Edit data/Create and save KML file/CreateAndSaveKML.storyboard

```diff
@@ -0,0 +1,78 @@
+                    <view key="view" contentMode="scaleToFill" id="HeE-Oi-vs0">
+                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="akX-0L-bf0" userLabel="Map View" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="0.0" width="375" height="623"/>
+                                <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="WEi-Qs-afR">
+                                <rect key="frame" x="0.0" y="623" width="375" height="44"/>
+                                <items>
+                                    <barButtonItem title="Reset" id="VbR-tF-dZd" userLabel="Reset">
+                                        <connections>
+                                            <action selector="resetKML" destination="XDb-RM-FgI" id="N7T-PU-Tcn"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="uYO-iU-ujY"/>
+                                    <barButtonItem systemItem="add" id="V8L-U8-tQY">
+                                        <connections>
+                                            <action selector="addFeature" destination="XDb-RM-FgI" id="Xyb-6T-rhz"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem systemItem="done" id="4d8-3n-68D">
+                                        <connections>
+                                            <action selector="completeSketch" destination="XDb-RM-FgI" id="i9k-ux-qU9"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="ZXj-oh-FBG"/>
+                                    <barButtonItem enabled="NO" systemItem="action" id="MuI-Qf-AEu">
+                                        <connections>
+                                            <action selector="saveKMZ:" destination="XDb-RM-FgI" id="U0V-fV-qEc"/>
+                                        </connections>
+                                    </barButtonItem>
```

Comment:
The way you have it is fine, but if you wanted to, you could move the bar button items outside of the toolbar and get rid of the flexible space items since they're going to be added to the toolbar in code. 

<img width="233" alt="image" src="https://user-images.githubusercontent.com/2257493/79484279-3f469b00-7fc8-11ea-8b5e-0f65a6677386.png">